src/ros/mb_publisher.cpp

```diff
@@ -148,15 +207,19 @@ bool MBPublisher::consume(MasterBoardData_V3_2& data)
 bool MBPublisher::consume(RobotModeData_V1_X& data)
 {
   publishRobotStatus(data);
+  publishRobotModeData(data);
   return true;
 }
 bool MBPublisher::consume(RobotModeData_V3_0__1& data)
 {
   publishRobotStatus(data);
+  publishRobotModeData(data);
   return true;
 }
 bool MBPublisher::consume(RobotModeData_V3_2& data)
 {
   publishRobotStatus(data);
+  // TODO: add robot mode v3.2 handler (would require changes to RobotModeData
```

Comment:
Let me add that by adding `publishRobotModeData(data)` here, I can see the messages being published.

Reply:
I could add that call to all `consume(..)` overloads. It wouldn't parse any of the additional fields, but it would cause publication of the data that is parsed for all supported versions.

Reply:
That call is already there for versions other than 3.2. That's why I wondered why you weren't adding it to this version as well.

Reply:
This was an oversight, not sure what (or whether) I was (still) thinking (t)here.